go-controller/pkg/cni/utils.go

```diff
@@ -131,3 +131,40 @@ func PodAnnotation2PodInfo(podAnnotation map[string]string, checkExtIDs bool, po
 	}
 	return podInterfaceInfo, nil
 }
+
+//START taken from https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/types/pod_update.go
+const (
+	ConfigSourceAnnotationKey    = "kubernetes.io/config.source"
+	ConfigMirrorAnnotationKey    = kapi.MirrorPodAnnotationKey
+	ConfigFirstSeenAnnotationKey = "kubernetes.io/config.seen"
+	ConfigHashAnnotationKey      = "kubernetes.io/config.hash"
+)
+
+const (
+	// Filesource idenitified updates from a file.
+	FileSource = "file"
+	// HTTPSource identifies updates from querying a web page.
+	HTTPSource = "http"
+	// ApiserverSource identifies updates from Kubernetes API Server.
+	ApiserverSource = "api"
```

Comment:
same here do you want all? only ApiserverSource right? and you can combine the two constants in the same block